src/DynamoCoreWpf/NodeViewCustomization/NodeViewCustomizationLoader.cs

```diff
@@ -100,21 +108,11 @@ public static Type MostDerivedCommonBase(IEnumerable<Type> types)
             //return types.First().TypeHierarchy().First(t => counts[t] == total);
         }
 
-        private static bool ImplementsGeneric(Type generic, Type toCheck)
-        {
-            var cur = toCheck.IsGenericType ? toCheck.GetGenericTypeDefinition() : toCheck;
+       
 
-            var isGenInterf = cur.GetInterfaces().Any(
-                x =>
-                    x.IsGenericType &&
-                        x.GetGenericTypeDefinition() == generic);
-
-            if (generic == cur || isGenInterf)
-            {
-                return true;
-            }
-
-            return false;
+        internal static bool ContainsNodeViewCustomizationType(Assembly assem)
```

Comment:
Same question here - can we reuse the method of the same name defined in `NodeModelAssemblyLoader`?